Examples/Windpower Example/windpower-example.py

```diff
@@ -0,0 +1,85 @@
+import json
+import PySAM.Windpower as wp
+
+# create a new windpower model
+wind = wp.new()
+
+# set model inputs from JSON generated by SAM
+with open('windpower-inputs.json', 'r') as file:
+    sam_data = json.load(file)
+    # loop through each key-value pair
+    for k, v in sam_data.items():
+        if k != 'number_inputs':
+            wind.value(k, v)
+
+## replace some inputs as needed for this analysis ##
+
+# load a wind resource file
+wind.value(	'wind_resource_filename', 'wind-resource-file.csv')
+
+# wind shear is location-specific to adjust wind speed data to turbine hub height
+wind.value('wind_resource_shear', 0.14)
+
+# load wind turbine parameters from JSON
+with open('wind-turbines.json', 'r') as file:
```

Comment:
same with this file path

Reply:
Fixed with 62f40619128218a81b30029a22a925f2dfb43a5d